projects/wallet-template/src/containers/WalletPopup/pages/Accounts/SwitchAccount.tsx

```diff
@@ -0,0 +1,85 @@
+import { User } from "lucide-react"
+import React, { useEffect, useState } from "react"
+import useSWR from "swr"
+import { rpc } from "../../api"
+import { RadioGroup } from "@headlessui/react"
+import { SubmitHandler, useForm } from "react-hook-form"
+import { useNavigate } from "react-router-dom"
+
+export const SwitchAccount: React.FC = () => {
+  const navigate = useNavigate()
+
+  const { data: keysets } = useSWR(
+    ["/rpc/keysets"],
+    () => rpc.client.listKeysets(),
+    { revalidateOnFocus: true },
+  )
+  const { data: primaryKeysetName, mutate } = useSWR(
+    ["/rpc/keysets/primaryName"],
+    () => rpc.client.getPrimaryKeysetName(),
+    { revalidateOnFocus: true },
+  )
+
+  const [selectedKeysetName, setSelectedKeysetName] = useState<string>("")
+  const { handleSubmit } = useForm()
+
+  useEffect(() => {
+    if (primaryKeysetName) {
+      setSelectedKeysetName(primaryKeysetName)
+    }
+  }, [primaryKeysetName])
```

Comment:
this state seems to be redundant and computed from `primaryKeysetName`
Why not just use `primaryKeysetName`?